genre.rb

```diff
@@ -0,0 +1,17 @@
+require './item'
+
+class Genre
+  attr_reader :name
```

Comment:
It would be nice if you add the "id" as attr_reader too, because we will use this property later.
```suggestion
  attr_reader :name, :id
```